.github/workflows/CICD.yml

```diff
@@ -0,0 +1,21 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+    
+    runs-on: ubuntu-latest
+  
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        # node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+        
+    - run: npm install
+    - run: npm audit fix --force
+    - run: npm run build
+    - run: ls '/home/runner/work/SEMFIRE/SEMFIRE'
```

Comment:
I don't understand these directories here. Why it has two SEMFIRES? what files are these navigating to?
From the acceptance criteria, it says " commands to run the main (index file) and check if it runs." 
Is this '/home/runner/work/SEMFIRE/SEMFIRE' the same as the acceptance criteria?